src/app/application.cpp

```diff
@@ -557,6 +558,13 @@ int Application::exec(const QStringList &params)
 #endif // DISABLE_WEBUI
 #else
     m_window = new MainWindow;
+		UiTheme::initInstance();
```

Comment:
@jagannatharjun:  Wrong indentation.  Please replace tabs with spaces, here and below.